Nodejs/Product/Nodejs/Nodejs.csproj

```diff
@@ -1245,6 +1245,10 @@
     <PackageReference Include="VSLangProj80">
       <Version>8.0.50728</Version>
     </PackageReference>
+    <PackageReference Include="Microsoft.SourceLink.GitHub">
```

Comment:
Should the reference be added to all projects? For example TestAdapter.csproj or Npm.csproj

Reply:
It should be added to anything we would expect to step into from other code. I think test assemblies are the classic example of what shouldn't be included, but everything else is probably fair game.

I guess we should also be publishing the pdbs for these? I think we currently only archive `NodeJsTools`.

Reply:
In the current build I see
`NodejsTools.pdb`
`NodejsTools.Npm.pdb`
`NodejsTools.PressAnyKey.pdb`
`NodejsTools.ProjectWizard.pdb`
`NodejsTools.TestAdapter.pdb`
`NodejsTools.TestAdapterImpl.pdb`
`JavaScript.TestAdapter.pdb`
`NodejsTools.WebRole.pdb`

Do you think it's reasonable to index and publish symbols for all of these?